store/ssmstore.go

```diff
@@ -40,15 +42,17 @@ type SSMStore struct {
 }
 
 // NewSSMStore creates a new SSMStore
-func NewSSMStore(numRetries int) (*SSMStore, error) {
+func NewSSMStore(numRetries int, minThrottleDelay time.Duration) (*SSMStore, error) {
 	ssmSession, region, err := getSession(numRetries)
 	if err != nil {
 		return nil, err
 	}
 
+	retryer := client.DefaultRetryer{NumMaxRetries: numRetries, MinThrottleDelay: minThrottleDelay}
+
 	svc := ssm.New(ssmSession, &aws.Config{
-		MaxRetries: aws.Int(numRetries),
```

Comment:
Setting max retries on the retryer instead of config because config document notes max retries is ignored if retryer and it are specified.

Reply:
Link to doc?

Reply:
https://docs.aws.amazon.com/sdk-for-go/api/aws/#Config

Thank you for prompting the second look. In rereading it the top-level MaxRetries option takes precedence. Just setting on the retryer does preserve behavior but my original comment is incorrect.